src/main/java/pl/cleankod/exchange/core/service/AccountService.java

```diff
@@ -0,0 +1,11 @@
+package pl.cleankod.exchange.core.service;
+
+import pl.cleankod.exchange.core.domain.Account;
+
+public interface AccountService {
+
+    Account execute(final Account.Id id);
```

Comment:
I don't understand what this service is supposed to do by looking at the class and methods names. How can we improve this?